docs/developers/tools-infra/thirdweb.md

```diff
@@ -0,0 +1,12 @@
+---
+title: thirdweb
+---
+
+# What is thirdweb
+
+## Resources
+
+Learn more and start building with thirdweb!
+
+- Learn [how to use thirdweb on Linea](/blog/thirdweb)
```

Comment:
Suggestion:
Maybe change the title to "Deploy your first smart contract with thirdweb"